### DISCUSSION TITLE
Account deletions error

### DISCUSSION THREAD

**Discussion by gitfiro** [QualiTorque/qualitorque.github.io#225] [Q&A] [answered]
I am trying to delete my account but it's failing to delete the cloud account, Even though I don't have an active sandbox in the trail. 
It asks me to end all sandboxes before deleting this account before deleting.

**Comment by Chris530906** [comment]
Hello @gitfiro ,

Sometimes environments that are in a bad state get hidden by the UI. When you are looking at the environment view, can you please verify that the "Active environments only" toggle is not active?

![image](https://user-images.githubusercontent.com/73956778/215800280-7b2a6ab2-6839-47e6-91d2-193fde666185.png)

If you can also provide the account name, we can investigate on our side. You can find the account name by going to the Admin section (Bottom Left) -> Account & Billing (Left Side) -> Account Details.

![image](https://user-images.githubusercontent.com/73956778/215802035-6b84b34c-1701-4cd8-b698-e9ec28fd76e8.png)

Please let us know if you have any other questions or issues.
-Chris

**Comment by gitfiro** [comment]
Hi Chris,

Thank you for emailing back. The problem is that I don't have active space
or a sandbox. Maybe can you heck it out please.

here is my account:  trial-f2297733

Thank you
Ramadan

**Comment by Chris530906** [comment, marked as answer]
Hey Ramadan,

Thank you for your patience. The account should now show as deleted, please let me know if that is not the case. Have a great rest of the week!

-Chris

@gitfiro

**Reply by redperalta23** [reply]
Hi Chris, can you also help me please? I have a CML that still says active status even if that lab has already ended days ago. I can't even end it. I get an error that says "Running terminate is not allowed at the moment, because there’s a conflict with currently running operations. Please wait until these operations are completed." whenever I try to end it. I don't see any running operations because it is the only lab that is in open status. By the way, this CML is just the free in devnetsandbox.
